samples/snippets/snippets_test.py

```diff
@@ -175,6 +192,17 @@ def wait_for_ready(client, key_version_name):
         time.sleep(0.1*(i**2))
     pytest.fail('{} not ready'.format(key_version_name))
 
+def test_check_state_import_job(project_id, location_id, key_ring_id, import_job_id):
+    check_state_import_job(project_id, location_id, key_ring_id, import_job_id)
```

Comment:
These tests don't look like they are actually checking anything. Can you capture the stdout and make sure the tests work as expected?

See https://github.com/googleapis/python-dlp/blob/a7b884e502fbbd8b941a8222d41f989ca392762c/samples/snippets/redact_test.py#L46